L1Trigger/L1THGCal/test/L1Clustering_taus_cfg.py

```diff
@@ -0,0 +1,186 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
Configuration file to be removed. It is not necessary to have one configuration file in releases for each type of particle.